starter/package.json

```diff
@@ -139,8 +139,8 @@
   },
   "devDependencies": {
     "@compodoc/compodoc": "~1.1.7",
-    "@nationalbankbelgium/stark-build": "latest",
-    "@nationalbankbelgium/stark-testing": "latest",
+    "@nationalbankbelgium/stark-build": "file:../dist/packages-dist/stark-build/nationalbankbelgium-stark-build-10.0.0-beta.4-f331991e.tgz",
+    "@nationalbankbelgium/stark-testing": "file:../dist/packages-dist/stark-testing/nationalbankbelgium-stark-testing-10.0.0-beta.4-f331991e.tgz",
```

Comment:
Can you revert these 2 changes back to "latest"? We should always keep it that way to ensure that whenever someone gets the Starter it will install the latest Stark ;)